testng-core/src/main/java/org/testng/internal/ITestClassConfigInfo.java

```diff
@@ -24,4 +29,22 @@ default List<ITestNGMethod> getAllBeforeClassMethods() {
   default List<ITestNGMethod> getInstanceBeforeClassMethods(Object instance) {
     return Lists.newArrayList();
   }
+
+  default List<ITestNGMethod> getInstanceAfterClassMethods(Object instance) {
+    return Lists.newArrayList();
+  }
+
+  static List<ITestNGMethod> allBeforeClassMethods(ITestClass tc) {
```

Comment:
Here, I have a conflict with 2 style rules:
* Use positive condition when possible
* Use quick exit for process check like inputs

I feel the second rule is more important. Wdyt?

Reply:
Sure. But in this case, either of them does not matter because it does not add to the code readability nor does it reduce the code complexity. 

I usually use the exit fast (the second rule) in situations wherein I can reduce the code complexity a lot.  Do you still want me to flip this?

Reply:
I agree. As we have not yet wroten code guideline, both are valid.